### PR TITLE
Map sound things, tile renderers, and other misc things.

### PR DESCRIPTION
**Pull request by darkevilmac**
Feel free to leave feedback, notable changes are mapping of Sounds for reference, and Sound which includes sound registration.

TileEntityRenderer has also been mapped which includes it's implementations.

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -135,6 +135,7 @@ CLASS none/akw net/minecraft/block/Block
 	METHOD a getRawIdFromBlock (Lnone/akw;)I
 		ARG 0 block
 	METHOD a (Lnone/akw;Lnone/akw;)Z
+	METHOD a setSounds (Lnone/apu;)Lnone/akw;
```
Comment (asiekierka) [inline mappings/net/minecraft/block/Block.mapping]: Are you sure "setSounds" is good? I don't like it. Perhaps "setSoundType"? "setSoundSet"? @shadowfacts

File: mappings/net/minecraft/block/impl/BlockFire.mapping [hunk 2]
```diff
@@ -7,10 +7,12 @@ CLASS none/amp net/minecraft/block/impl/BlockFire
 	FIELD f UP Lnone/asv;
 	METHOD a deserializeState (I)Lnone/asj;
 	METHOD a getDropAmount (Ljava/util/Random;)I
+	METHOD a setBurnLevels (Lnone/akw;II)V
```
Comment (asiekierka) [inline mappings/net/minecraft/block/impl/BlockFire.mapping]: setBurnLevel*s*? Please define the parameters so we can decide on the name further.

File: mappings/net/minecraft/block/impl/BlockFire.mapping [hunk 3]
```diff
@@ -7,10 +7,12 @@ CLASS none/amp net/minecraft/block/impl/BlockFire
...
 	METHOD a getCollisionBox (Lnone/asj;Lnone/aiy;Lnone/cn;)Lnone/bcp;
 	METHOD b createStateFactory ()Lnone/ask;
 	METHOD b randomTick (Lnone/aiu;Lnone/cn;Lnone/asj;Ljava/util/Random;)V
 	METHOD c getRenderingState (Lnone/asj;Lnone/aiy;Lnone/cn;)Lnone/asj;
+	METHOD e init ()V
```
Comment (asiekierka) [inline mappings/net/minecraft/block/impl/BlockFire.mapping]: This doesn't init BlockFire, it inits burn values - initX, where X is the renamed setBurnLevels' latter half, would fit better.

File: mappings/net/minecraft/sound/BlockSound.mapping [hunk 4]
```diff
@@ -0,0 +1,29 @@
+CLASS none/apu net/minecraft/sound/BlockSound
```
Comment (asiekierka) [inline mappings/net/minecraft/sound/BlockSound.mapping]: BlockSound? see setSounds comment above

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (shadowfacts) [inline mappings/net/minecraft/block/Block.mapping]: Depends what the class itself is called. If the class is called Sounds setSounds is fine, if the class is call SoundType then it should be setSoundType

**Comment by darkevilmac** [comment]
@asiekierka Fixed mentioned issues.

**Pull request merged**